configure.ac

```diff
@@ -5933,6 +5933,10 @@ else
 fi
 AC_SUBST(TEST_MODULES)
 
+# yk integration
+AC_CHECK_HEADER([yk.h], [], [AC_MSG_ERROR([yk installation not found])])
```

Comment:
Can we make the errors more descriptive.

"yk.h header not found" and "libykcapi library not found"

At the moment a user wouldn't be able to differentiate the two problems.

Reply:
Those are pretty standard `configure` errors -- and, I must admit, I don't know how they could be better without becoming ludicrously long. I'm open to suggestions though.

Reply:
> "yk.h header not found" and "libykcapi library not found"

Reply:
configure will say something like:

```
looking for yk.h... yk installation not found
```

if it can't find it. Does it help to repeat "yk.h" in the error. I can just `exit 1` or whatever.

Reply:
OK, I didn't realise that it was already printed.